Automattic/wp-calypso
# Framework: `dispatchRequest` update (sites users select)

#### Changes proposed in this Pull Request

See #25121

In this patch we're replacing the use of `dispatchRequest()` in the data layer handler to use the newer API exposed as `dispatchRequestEx()` This should have no change in actual effect or interaction.

(I still need to finish updating the Jest tests, this is a WIP.)

#### Testing instructions

* Switch to this PR, navigate to My Sites -> People 
* Make sure no errors are thrown in the console
* Make sure Jest tests pass
* ...? I'm not sure how else to verify this is working properly. Guidance appreciated!
